src/makeRelativeDate.README.md

```diff
@@ -0,0 +1,23 @@
+
+# `makeRelativeDate`
+
+Takes an absolute date in epoch format and returns a string representing relative time ago.
+
+## Props
+```typescript
+type FormatType = 'short' | 'med' | 'long';
+
+type RelativeDateOptions = {
+	notAfter?: number; // Return false if the date ago is past a cutoff
```

Comment:
Does the date have to be _before_ this timestamp to return a `string` instead of `false`? Ah, it’s not a timestamp, but rather a number of seconds.

Should this be called `notFurtherThan`?

Reply:
how about `maxAge`?

Reply:
(named after you @mxdvl)

Reply:
I actually agree, this name is confusing. However, I'm lifting and shifting this code from DCR which was itself lifted from Frontend and I'm concious that if we ever were to complete the circle and import this into Frontend it would be a lot easier to that work if the naming conventions of such properties had remained the same

Reply:
I went with `maxAge` in honour of @mxdvl

Reply:
in this instance, by paying off a little bit of technical debt we'd get the benefit of a cleaner API going forward (it should be pretty easier to search and replace?). 

the alternative is we invest in future technical debt by carrying over a more confusing API into future projects, which seems like a bad move to me?

(also, in frontend [there are zero instances of `notAfter` actually being used](https://github.com/guardian/frontend/search?q=notAfter)...)

Reply:
If `notAfter` isn't being used in Frontend, and it isn't being used in DCR, and no-one _really_ understands what it's for, we should just get rid of it?

We can revisit if things change but this doesn't seem like an edge case we need to future proof for.

Reply:
Agreed, there's no point shipping code no one uses. Maybe it was once useful ¯\_(ツ)_/¯

Reply:
Sorry @mxdvl , I've removed your namesake property. Fame is fleeting!